src/main/java/mate.academy/Main.java

```diff
@@ -6,12 +6,15 @@
 import mate.academy.service.ProductService;
 
 public class Main {
-
     public static void main(String[] args) {
-        // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
-        List<Product> products = productService.getAllFromFile("products.txt");
-        products.forEach(System.out::println);
+        try {
+            ProductService productService = (ProductService) injector
+                    .getInstance(ProductService.class);
+            List<Product> products = productService.getAllFromFile("products.txt");
+            products.forEach(System.out::println);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
```

Comment:
The catch block is catching a generic Exception, which is not recommended. It is better to catch specific exceptions that might be thrown. Also, when rethrowing an exception, it's important to include additional context information and the original exception to aid in debugging. Consider using a more specific exception and providing a more informative error message.